tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -175,7 +172,7 @@ spec:
           value: openstack.octavia-kuttl-tests.svc
         - name: DatabaseName
           value: octavia
-        - name: DatabaseUser
+        - name: databaseAccount
```

Comment:
This is an environment var that is passed to the sync job script and didn't change names